sonic_platform_base/sonic_xcvr/mem_maps/amphenol/backplane.py

```diff
@@ -0,0 +1,18 @@
+"""
+    Custom memory map for Amphenol 800G Backplane connector
+"""
+
+from ...mem_maps.public.cmis import CmisFlatMemMap
+from ...fields.xcvr_field import (
+    NumberRegField,
+    RegGroupField,
+    CodeRegField
+)
+from ...fields import consts
+
+class AmphBackplaneMemMap(CmisFlatMemMap):
```

Comment:
@prgeor Just for my information, is this a flat memory module?

Reply:
@mihirpat1 yes